oscarlab/graphene
# [Pal/Linux-SGX] sgx_sign.py: Use proper TOML parser for manifests

## Description of the changes 

This PR is a preparation for another one, which will resolve #2205.

The changes are:
* Changing sgx_sign.py to use a proper TOML parsing library, not some handcrafted string shuffling.
* Fixing a small bug in trusted files handling, where a missing/non-matching `sgx.trusted_checksum` entry resulted in treating the file as allowed.

## How to test this PR? 

CI should be enough. For the `sgx.trusted_checksum` issue you need to modify `sgx_sign.py` to make it generate invalid entries.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2210)
